src/main.py

```diff
@@ -20,18 +25,20 @@ def main():
         f.close()
 
     try:
-        process = CrawlerProcess()
-        process.crawl(CisaSpider) 
-        process.crawl(DgssiSpider)
-        process.crawl(CertFrSpider)
+        crawler_settings = Settings()
+        crawler_settings.setmodule(cyberowl_settings)
+        process = CrawlerProcess(settings=crawler_settings)
+        process.crawl(CisaSpider)
+        process.crawl(MACertSpider)
+        process.crawl(CertFRSpider)
         process.crawl(IBMCloudSpider)
         process.crawl(ZDISpider)
         process.crawl(VigilanceSpider)
         process.crawl(VulDBSpider)
         process.start()
 
-    except Exception:
-        raise ValueError("Error in the spiders!")
+    except Exception as exc:
+        raise ValueError("Error in the spiders!") from exc
```

Comment:
Can you please explain what `from exc` do here?

Reply:
`from exc` here shows the cause of the exception; as if, each time this exception is raised it also shows the exceptions that led to it. 
However, I am still trying to find a way to implement it properly, because it currently seems unnecessary.